Mathlib/MeasureTheory/OuterMeasure/AE.lean

```diff
@@ -106,15 +106,25 @@ theorem ae_ball_iff {ι : Type*} {S : Set ι} (hS : S.Countable) {p : α → ∀
     (∀ᵐ x ∂μ, ∀ i (hi : i ∈ S), p x i hi) ↔ ∀ i (hi : i ∈ S), ∀ᵐ x ∂μ, p x i hi :=
   eventually_countable_ball hS
 
-theorem ae_eq_refl (f : α → β) : f =ᵐ[μ] f :=
-  EventuallyEq.rfl
+lemma ae_eq_refl (f : α → β) : f =ᵐ[μ] f := EventuallyEq.rfl
+lemma ae_eq_rfl {f : α → β} : f =ᵐ[μ] f := EventuallyEq.rfl
```

Comment:
Why this duplication? Could one be deprecated?

Reply:
We have this duplication for every concrete reflexive relation, eg `le_refl`/`le_rfl`. I'm not sure this PR is the place to upturn such a widespread pattern and naming convention.

Reply:
:-( This makes me really sad, but okay.